gradle/libs.versions.toml

```diff
@@ -28,8 +28,15 @@ agp = "7.2.1"
 lint = "30.2.1"
 
 # Changelog: https://github.com/JetBrains/kotlin/blob/master/ChangeLog.md
-# @see kotlin_version in buildSrc/gradle.properties
-kotlin = "1.4.32"
+# Actual Kotlin version for dependencies. It has to be older to make sure AGP 3.x is still compatible.
+kotlin-target = "1.4.32"
+# Actual Kotlin language and API version for compilcation, it restricts language features and metadata.
+kotlin-language = "1.4"
+# Version of Kotlin Gradle Plugin used for compilation and stuff.
+# Note: this has to be in sync with Gradle's embedded Kotlin version.
```

Comment:
does it though?

Reply:
https://github.com/TWiStErRob/net.twisterrob.gradle/pull/251/commits/e5ab128f9b0ea78f286bfd8e9b757c7ebffd4c32